File: src/main/java/com/qtech/bubbles/common/logging/old/GameLoggingLevels.java
package com.qtech.bubbles.common.logging.old;

import java.util.logging.Level;

public class GameLoggingLevels extends Level {
    public static final Level DEBUG = new GameLoggingLevels("DEBUG", 1);

    @SuppressWarnings("SameParameterValue")
    protected GameLoggingLevels(String name, int value) {
        super(name, value);
    }

    protected GameLoggingLevels(String name, int value, String resourceBundleName) {
        super(name, value, resourceBundleName);
    }
}

File: src/main/kotlin/com/qtech/bubbles/common/logging/old/GameLoggingLevels.kt
package com.qtech.bubbles.common.logging.old

import java.util.logging.Level

open class GameLoggingLevels : Level {
    protected constructor(name: String?, value: Int) : super(name, value) {}
    protected constructor(name: String?, value: Int, resourceBundleName: String?) : super(name, value, resourceBundleName) {}

    companion object {
        val DEBUG: Level = GameLoggingLevels("DEBUG", 1)
    }
}